app/uk/gov/hmrc/helptosavefrontend/views/reminder/reminder_dashboard.scala.html

```diff
@@ -38,14 +38,14 @@ <h1 class="govuk-heading-l">@messages("hts.reminder-dashboard.h2")</h1>
           s"""${messages("hts.reminder-dashboard.t1-1")} ${PeriodUtils.getMessage(period)} ${messages("hts.reminder-dashboard.t1-2")}""".stripMargin,
           routes.ReminderController.getSelectedRendersPage,
           messages("hts.reminder-dashboard.t1"),
-          Some(messages("hts.reminder-dashboard-change-cancel"))
+          (messages("hts.reminder-dashboard-change-cancel"))
         ),
         summaryRowHelper.summaryListRow(
           messages("hts.reminder-dashboard.t2"),
           email,
           routes.AccountHolderController.getUpdateYourEmailAddress,
           messages("hts.reminder-dashboard.t2"),
-          Some(messages("hts.reminder-dashboard-change"))
+          (messages("hts.reminder-dashboard-change"))
```

Comment:
same as previous call to this helper, the param order has changed but these params are in the wrong order when parameter names are missing - use parameter names e.g. `changeLabel = Some(messages("hts.reminder-dashboard-change-cancel"))`

Reply:
reverted in [887c29b](https://github.com/hmrc/help-to-save-frontend/pull/1072/commits/887c29b3776f4dd7390c1270956b721cb50b76b5)